Tasks/AzureAppConfigurationImportV10/task.json

```diff
@@ -2,14 +2,14 @@
     "id": "fd186364-adb5-43c0-91ea-c5f9daa5b747",
     "name": "AzureAppConfigurationImport",
     "friendlyName": "Azure App Configuration Import",
-    "description": "Import key-values to an Azure App Configuration instance",
+    "description": "Import key-values to an Azure App Configuration instance in your build or deployment pipelines",
```

Comment:
There are no key-values in pipelines to import. How does this sound to you?

Import settings from configuration files into Azure App Configuration through build or deployment pipelines.